README.md

```diff
@@ -85,10 +85,15 @@ Explain what I have done and why?
 
 Import all my graphs and charts to Github
 
-## Processes
+## Processes 
 
-Image of kanban board
-What I used 
+The basic Kanban board from GitHub Projects was used for Project Management 👇
+
+![Screenshot from GitHub Projects](Kanban.png)
+
+### Why did I use a Kanban board?
+
+A kanban board is an agile project management tool designed to help visualize work, limit work-in-progress, and maximize efficiency. The reason I choose to use a Kanban board is because it allowed me to order my project tasks into sections such as; "To Do, In Progress, In Review and Done." This was very beneficial because I knew what needed to be done first and what I was waiting on a response for and could leave for now. It also allowed me to see how long I was spending on a task and allow me to manage my time better because I understood I only had 3 weeks to complete my final project and a set amount of tasks to do before I was done.
```

Comment:
```suggestion
A kanban board is an agile project management tool designed to help visualize work, limit work-in-progress, and maximize efficiency. I chose to use a Kanban board is because it allowed me to order my project tasks into sections such as; "To Do, In Progress, In Review and Done." This was very beneficial because I knew what needed to be done first and what I was waiting on a response for and could leave for now. It also allowed me to see how long I was spending on a task and allow me to manage my time better because I understood I only had 3 weeks to complete my final project and a set amount of tasks to do before I was done.
```